cli/sync_service/README.md

```diff
@@ -0,0 +1,362 @@
+# Model Management System
+
+## Introduction
+The model management system (MMS) is one of the most important parts in Edge Fabric. It is leveraged to ease the burden of AI model management of cognitive services running on an edge node. Edge Fabric provides a set of hzn CLIs to use MMS to manipulate the models object and its metadata.
```

Comment:
We need more introductory material here. We need to put this in context of the overall IEC product.